ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml

```diff
@@ -110,6 +110,14 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
     workflow: openshift-e2e-aws-single-node
+- as: e2e-aws-single-node-serial
+  cron: 0 0 * * *
```

Comment:
```suggestion
  interval: 24h
```

Reply:
Want to keep it consistent with the [other](https://github.com/openshift/release/blob/7cc0b2d767328e6667627d306bbce6dec381f10a/ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml#L106-L125) SNO jobs without modifying them (I'm afraid modifying them from 0 0 * * * to 24h might change the exact time they run daily, I don't know how it works)

Reply:
Let me know if you still insist I change this :)

Reply:
I think that we switched the other jobs to `interval` today, too ;)

Reply:
https://github.com/openshift/release/pull/19866

Reply:
What are the odds... Fixed & force pushed